src/Stratis.Bitcoin.Features.Api/ApiFeature.cs

```diff
@@ -117,29 +119,42 @@ public override void Dispose()
         }
     }
 
-    public sealed class ApiFeatureOptions
+    public sealed class ApiSettingsDefaults
     {
+        /// <summary>The default port used by the API when the node runs on the Stratis network.</summary>
+        public string ApiHost { get; set; }
+        public Network Network { get; set; }
```

Comment:
I don't think Network should be here. Whatever consumes it off this object should be obtaining it from somewhere else.

Reply:
As a general pattern for settings classes, passing in the network allows the initial defaults to be constructed based on the type of network or properties available on the network. Passing via the constructor will cause the user having to update his code when properties are added. The user should instead set properties by the following pattern:
```
var defaults = new ApiSettingsDefaults(network) {
    <Perform overriding of defaults here>
};
```

Reply:
I agree with @rowandh, we don't need to keep a reference to Network here 👍

Reply:
This will be fixed in the next PR which will pass a `StringBuilder` to the `PrintHelp` method, so that the latter would not require access to `Network` via `ApiDefaultSettings`.